core/domain/opportunity_services.py

```diff
@@ -189,8 +189,37 @@ def add_new_exploration_opportunities(story_id, exp_ids):
     """
     story = story_fetchers.get_story_by_id(story_id)
     topic = topic_fetchers.get_topic_by_id(story.corresponding_topic_id)
-    explorations = exp_fetchers.get_multiple_explorations_by_id(exp_ids)
+    _create_exploration_opportunities(story, topic, exp_ids)
+
+
+def create_exploration_opportunities_for_story_and_topic(story, topic):
+    """Creates new exploration opportunities corresponding to the supplied
+    story and topic.
+
+    Args:
+        story: Story. The story domain object corresponding to the exploration
+            opportunities.
+        topic: Topic. The topic domain object corresponding to the exploration
+            opportunities.
+    """
+    exp_ids_in_story = story.story_contents.get_all_linked_exp_ids()
```

Comment:
Can we move this step inside `_create_exploration_opportunities` and avoid having extra args?